README.md

```diff
@@ -1,276 +1,3 @@
-    score, acc = model.evaluate(x_test, y_test, show_accuracy=True, verbose=0)
-    print('Test accuracy:', acc)
-    return {'loss': -acc, 'status': STATUS_OK, 'model': model.to_yaml()}
-```
-
-Once the basic setup is defined, running the minimization is done in just a few lines of code:
-
-```python
-from elephas.hyperparam import HyperParamModel
-from pyspark import SparkContext, SparkConf
-
-# Create Spark context
-conf = SparkConf().setAppName('Elephas_Hyperparameter_Optimization').setMaster('local[8]')
-sc = SparkContext(conf=conf)
-
-# Define hyper-parameter model and run optimization
-hyperparam_model = HyperParamModel(sc)
-hyperparam_model.minimize(model=model, data=data, max_evals=5)
-```
-
-## Distributed training of ensemble models
-
-Building on the last section, it is possible to train ensemble models with elephas by means of running hyper-parameter 
-optimization on large search spaces and defining a resulting voting classifier on the top-n performing models. 
-With ```data``` and ```model``` defined as above, this is a simple as running
-
-```python
-result = hyperparam_model.best_ensemble(nb_ensemble_models=10, model=model, data=data, max_evals=5)
-```
-In this example an ensemble of 10 models is built, based on optimization of at most 5 runs on each of the Spark workers.
-
-## Discussion
-
-Premature parallelization may not be the root of all evil, but it may not always be the best idea to do so. Keep in 
-mind that more workers mean less data per worker and parallelizing a model is not an excuse for actual learning. 
-So, if you can perfectly well fit your data into memory *and* you're happy with training speed of the model consider 
-just using keras.
-
-One exception to this rule may be that you're already working within the Spark ecosystem and want to leverage what's 
-there. The above SparkML example shows how to use evaluation modules from Spark and maybe you wish to further process 
-the outcome of an elephas model down the road. In this case, we recommend to use elephas as a simple wrapper by setting 
-num_workers=1.
-
-Note that right now elephas restricts itself to data-parallel algorithms for two reasons. First, Spark simply makes it 
-very easy to distribute data. Second, neither Spark nor Theano make it particularly easy to split up the actual model 
-in parts, thus making model-parallelism practically impossible to realize.
-
-Having said all that, we hope you learn to appreciate elephas as a pretty easy to setup and use playground for 
-data-parallel deep-learning algorithms.
-
-
-## Literature
-[1] J. Dean, G.S. Corrado, R. Monga, K. Chen, M. Devin, QV. Le, MZ. Mao, M’A. Ranzato, A. Senior, P. Tucker, K. Yang, and AY. Ng. [Large Scale Distributed Deep Networks](http://research.google.com/archive/large_deep_networks_nips2012.html).
-
-[2] F. Niu, B. Recht, C. Re, S.J. Wright [HOGWILD!: A Lock-Free Approach to Parallelizing Stochastic Gradient Descent](http://arxiv.org/abs/1106.5730)
-
```

Comment:
Can you please not delete the whole readme? :D Just add your suggestion on how to run a Spark submit job somewhere it suits, ok?